src/pages/console/index.mdx

```diff
@@ -7,10 +7,29 @@ export const meta = {
 
 AWS Amplify Studio is a visual development environment for building fullstack web and mobile apps. Studio builds on existing backend building capabilities in AWS Amplify, allowing you to accelerate your UI development as well. With Studio, you can quickly build an entire web app, front-to-back, with minimal coding, while still maintaining full control over your app design and behavior through code. Ship faster, scale effortlessly, and delight every user.
 
-<InternalLinkButton href="/console/tutorial/buildui">
+<InternalLinkButton href="/console/tutorial/start">
   <span slot="text">Tutorial</span>
 </InternalLinkButton>
 
+## Get started without an AWS account
+
+If you are new to AWS, you do not need an AWS account to get started. You can still model and test your data before deploying to the cloud. After building your data model you must connect an AWS account to deploy your backend environment to Amplify using Amplify Studio. The sandbox interface lets you perform the following tasks:
+ * Set up your data model. For instructions on creating models and setting relationships, see [Data modeling in Amplify Studio](/console/data/data-model#Data-modeling-in-the-Admin-UI).
+ * Test your new data model locally.
+ * Deploy your backend to the cloud. This requires an AWS account.
+
+<Card
+  external
```

Comment:
Should this be opening in a new tab?

Reply:
yes, this should open a new tab. reached out to documentation team on this bug.